.gitlab-ci.yml

```diff
@@ -76,11 +76,11 @@ before_script:
     name: "$CI_JOB_NAME"
     paths:
       - _install_ci
+      # All those are for the test-suite jobs, to be discarded soon
       - config/Makefile
       - config/coq_config.py
       - config/coq_config.ml
       - config/dune.c_flags
-      - dmesg.txt
```

Comment:
@ejgallego  ping. Remove this change and leave for future cleanup. We should remove it when we stop generating it as Enrico suggests.

Reply:
The diff is misleading, the behavior should be preserved